mixins/StoreMixin.js

```diff
@@ -163,6 +171,13 @@ var StoreMixin = {
 
     // forceUpdate if not suppressed by configuration
     if (listenerDetail.suppressUpdate !== true && typeof this.forceUpdate === 'function') {
+      if (process.env.NODE_ENV !== 'production') {
+        var warning = 'Forced upates are an antipattern. ';
+        if (this.saveState_key != null) {
+          warning += 'Check the render method of ' + this.saveState_key + '.';
+        }
+        console.warn(warning);
+      }
```

Comment:
To clarify, I am strongly of the opinion that using forceUpdate as often as we do is [a very bad idea](https://facebook.github.io/react/docs/component-api.html#forceupdate). If we highlight every forced update with a console warning, we'll keep ourselves honest about what's going on behind the comfortable abstraction of the StoreMixin. 

`You wouldn't eat the sausage if you'd seen it being made`